packages/vitest/src/node/cache/index.ts

```diff
@@ -2,6 +2,7 @@ import fs from 'node:fs'
 import { findUp } from 'find-up'
 import { resolve } from 'pathe'
 import { loadConfigFromFile } from 'vite'
+import { generateHash } from '@vitest/runner/utils'
```

Comment:
Runner package should not be used in node

Reply:
Ok, providing you are still ok with hashing the project name as a solution, I have switched this to use `node:crypto` to generate an md5 hash